features/checkout/shipping_order/preventing_not_available_shipping_method_selection.feature

```diff
@@ -5,51 +5,52 @@ Feature: Preventing not available shipping method selection
     I want to not be able to choose not available shipping methods
 
     Background:
-        Given the store operates on a single channel in "France"
+        Given the store operates on a single channel in "United States"
         And the store has a product "Targaryen T-Shirt" priced at "$19.99"
         And I am a logged in customer
 
     @ui
     Scenario: Not being able to select disabled shipping method
-        Given the store has "Raven Post" shipping method with "€10.00" fee
-        And the store has disabled "Dragon Post" shipping method with "€30.00" fee
+        Given the store has "Raven Post" shipping method with "$10.00" fee
+        And the store has disabled "Dragon Post" shipping method with "$30.00" fee
         And I have product "Targaryen T-Shirt" in the cart
         When I am at the checkout addressing step
-        And I specify the shipping address as "Ankh Morpork", "Frost Alley", "90210", "France" for "Jon Snow"
+        And I specify the shipping address as "Ankh Morpork", "Frost Alley", "90210", "United States" for "Jon Snow"
         And I complete the addressing step
         Then I should not be able to select "Dragon Post" shipping method
 
     @ui
     Scenario: Not being able to select shipping method not available for my shipping address
-        Given there is a zone "EU" containing all members of the European Union
+        Given there is zone "NA" containing all members of the North America
         And there is a zone "The Rest of the World" containing all other countries
-        And the store has "Dragon Post" shipping method with "€30.00" fee within "EU" zone
-        And the store has "Raven Post" shipping method with "€10.00" fee for the rest of the world
+        And the store ships to "Germany"
+        And the store has "Dragon Post" shipping method with "$30.00" fee for the rest of the world
+        And the store has "Raven Post" shipping method with "$10.00" fee within "NA" zone
```

Comment:
If you wouldn't change the zones for shipping methods we could use `United States` as shipping address and get rid of irrelevant step `store ships to Germany`